app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,13 @@
   </head>
 
   <body>
+    <% if user_signed_in? %>
+      <%= link_to "Log out", destroy_user_session_path, data: { turbo_method: :delete } %>
+    <% else %>
+      <%= link_to "Log in", new_user_session_path %>
+    <% end %>
+    <%= content_tag(:p, notice, class: 'notice', style: "color: green") if notice.present? %>
+    <%= content_tag(:p, alert, class: 'alert', style: "color: red") if alert.present? %>
```

Comment:
A place for flash messages (such as "x has been created"), both alerts and notices. Putting them in the layout means we don't have to keep putting them in each view. Note that scaffold adds them into the view.